src/transform/index.spec.js

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
In test-fixture metadata files add key "description" and as its value description of thing that this test tests